pom.xml

```diff
@@ -66,6 +66,11 @@
             <type>war</type>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apereo.cas</groupId>
```

Comment:
Yes it does. If you want to go with the alternate option (wget), I've tested and it works, however I'll need to know where you want the downloaded command-shell jar and are you ok with hard-coding the version for that inside the .sh/.bat files?